src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/on_app_interface_unregistered_notification.h

```diff
@@ -71,6 +71,12 @@ class OnAppInterfaceUnregisteredNotification
   virtual void Run();
 
  private:
+  /**
+   * @brief Check whether a connection to mobile must be closed after sending
+   * this message
+   **/
+  bool IsFinalMessage();
```

Comment:
@mvorobio Don't you want to make it `const` as it is a class member function?

Reply:
@AByzhynar please check https://github.com/LuxoftSDL/sdl_core/pull/18/commits/df8c18c6c1f540cb8fdd56fff93670282a59de09